merkletree/pad.go

```diff
@@ -78,6 +84,10 @@ func (pad *PAD) updateInternal(policies *Policies, epoch uint64) {
 	}
 }
 
+// Update updates the history hash chain by issuing
+// new signed tree root. It may remove some older STRs from
+// memory if the snapshot array exceeded its maximum capacity.
+// The Policies pointer can be nil if the policies does not change.
```

Comment:
I would start with more PAD-related terminology here: E.g. "Update generates a new snapshot of the directory. Specifically, it extends the hash chain by issuing..."